blazor/stock-chart/datetime-category-axis.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Events in Blazor Stock Chart Component | Syncfusion
+description: Checkout and learn here all about events in Syncfusion Blazor Stock Chart component and much more details.
+platform: Blazor
+control: Stock Chart 
+documentation: ug
+---
+
+# Date Time Category Axis in Blazor Stock Chart Component 
+
+Date Time Category Axis in Stock chart is used to display the minimum number of labels without any overlap on various business days.
```

Comment:
# DateTimeCategory Axis in Blazor Stock Chart Component

`DateTimeCategory` axis in the stockchart is used to display the minimum number of labels without any overlap on various business days. To use the date time category axis set the [ValueType](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ValueType.html) property as [DateTimeCategory](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ValueType.html#Syncfusion_Blazor_Charts_ValueType_DateTimeCategory)